codegen/projections/white_label/sig/white_label/client.rbs

```diff
@@ -182,5 +183,28 @@ module WhiteLabel
         ?member___next_token: ::String
       ) -> Hearth::Output[Types::Struct____PaginatorsTestWithBadNamesOutput]
 
+    def start_event_stream: (?::Hash[::Symbol, untyped] params, ?::Hash[::Symbol, untyped] options)  ?{ (::String) -> Hearth::BlockIO } -> Hearth::Output[Types::StartEventStreamOutput] |
```

Comment:
I haven't done any of the eventstreams generated RBS yet.  These shouldn't allow a block.  I'll fix these (potentially in a future PR to avoid extra complexity in this one).